### PR TITLE
db dir

### PR DESCRIPTION
**Pull request by MarinPostma**
This pr adds all the files created by sqld to a directory, instead of creating multiple standalone files.

It also renames the default db name from `iku.db` to `data.sqld`

### CODE REVIEW

**Comment by MarinPostma** [comment]
bors try

**Comment by penberg** [comment]
@MarinPostma This will make any existing deployment not start, right? What would be a good migration path?

File: sqld/src/main.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ use sqld::Config;
 #[command(name = "sqld")]
 #[command(about = "SQL daemon", long_about = None)]
 struct Cli {
-    #[clap(long, short, default_value = "iku.db", env = "SQLD_DB_PATH")]
+    #[clap(long, short, default_value = "data.sqld", env = "SQLD_DB_PATH")]
```
[the remark this answers is not in the document]
Comment (MarinPostma) [reply sqld/src/main.rs:12]: that's the goal, hide what it truly is

[the remark this answers is not in the document]
Comment (MarinPostma) [reply sqld/src/main.rs:12]: @penberg put everything in running databases into a `data.sqld` dir

**Review by penberg — APPROVED** [review]

**Comment by MarinPostma** [comment]
bors merge

**Comment by haaawk** [comment]
Why couldn't we have a version of sqld that can read both paths and rewrite the files from the old path to the new path if needed?

**Pull request merged**